### ISSUE TITLE
Items lack transcriptions

### ISSUE THREAD

**Issue by erinchambers** [CDRH/genoa_mukurtu#102]
Like some of our other projects, either some or all of the documents on the Genoa site exist only in image format and are untranscribed, which is a violation of the ADA Title II updated regulations that go into effect on April 26, 2027.

**Comment by erinchambers** [comment]
Tagging this for discussion with Andy J about FromthePage transcriptions
